competency-change.md

```diff
@@ -0,0 +1,13 @@
+Broad InnerSource adoption an example of a competency change.
+Some competency (in this example InnerSource) should be upleveled across the company.
+
+Here are the roles involved in running a competency change effort.
```

Comment:
```suggestion
Here are the roles involved in running a competency change effort (also called "competency program" below).
```

Not sure which term is best but I would suggest to only use one.
So maybe even replace "effort" with "program" here?